### PR TITLE
Simplify code with ts-is-kind

### PR DESCRIPTION
**Pull request by mohsen1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-    "devDependencies": {
-        "@types/jest": "^19.2.4",
-        "@types/node": "^7.0.31",
-        "styled-components": "^2.0.1",
-        "ts-jest": "20"
-    },
-    "files": [
-        "dist"
-    ]
+  "name": "typescript-plugin-styled-components",
```
Comment (mohsen1) [inline package.json]: `npm i --save` did it 😭

**Comment by Igorbek** [comment]
Not bad! let me review it more closely a bit later. Is that your library that is used?

**Comment by mohsen1** [comment]
Yes, it's just a big list of is* functions

**Comment by mohsen1** [comment]
wanna merge this?

File: package.json (hunk 1 above)
Comment (Igorbek) [inline package.json]: May I ask for fixing the indent to use 4 spaces and rebasing the changes. I don't really like the noise in the history.

Comment (Igorbek) [inline package.json]: May I ask for fixing the indent to use 4 spaces and rebase the changes. I don't really like the noise in the history.

**Comment by Igorbek** [comment]
Thank you for the contribution!

**Pull request merged**